recipe/build.sh

```diff
@@ -14,20 +14,30 @@ cmake ${CMAKE_ARGS} \
       -D CMAKE_INSTALL_PREFIX=${PREFIX} \
       -D CMAKE_INSTALL_LIBDIR=lib \
       -D EXE_SQLITE3=${EXE_SQLITE3} \
```

Comment:
The docs say that we should provide paths for `-DSQLITE3_INCLUDE_DIR` and `-DSQLITE3_LIBRARY` too:
```
If the SQLite3 dependency is installed in a custom location, specify the paths to the include directory and the library:

cmake -DSQLITE3_INCLUDE_DIR=/opt/SQLite/include -DSQLITE3_LIBRARY=/opt/SQLite/lib/libsqlite3.so ..
```
See https://proj.org/en/9.3/install.html#build-steps and https://proj.org/en/9.3/install.html#cmdoption-arg-SQLITE3_INCLUDE_DIR and https://proj.org/en/9.3/install.html#cmdoption-arg-SQLITE3_LIBRARY

Reply:
this is probably not needed. cmake can find the libraries in CMAKE_INSTALL_PREFIX and CMAKE_INSTALL_LIBDIR and consider that as a standard location.